### ISSUE TITLE
Symfony AutoMapperBundle requires `symfony/serializer` and `symfony/property-access`

### ISSUE THREAD

**Issue by GromNaN** [jolicode/automapper#127]
On a fresh installation of Symfony 7.1, once `AutoMapperBundle` (version 9.0.1) is enabled in `config/bundles.php`, I get this error:

> The service `automapper.mapping.class_discriminator_from_class_metadata` has a dependency on a non-existent service `serializer.mapping.class_metadata_factory`.

After installing the `symfony/serializer` package so that the service is registered by the `FrameworkBundle`, I get a new error:

> [`Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException`] You have requested a non-existent service `serializer.normalizer.object`.

This is solved by installing `symfony/property-access`. But then:

> [`Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException`] The service `automapper.property_info.cache.inner` has a dependency on a non-existent service `property_info.phpstan_extractor`.

This is solved by installing both `phpstan/phpdoc-parser` and `phpdocumentor/type-resolver`.

So I need to install this packages to work with the bundle:

```
composer require symfony/serializer symfony/property-access phpstan/phpdoc-parser phpdocumentor/type-resolver
```

This is the first time I install the package, so I'm not sure this is correct.

I think the installation of these dependencies could be better guided in the docs.

**Comment by Korbeil** [comment]
You are correct ! And we shouldn't force users to install Symfony's Serializer since it's now optional, I'll try to fix this.

**Comment by Korbeil** [comment]
Fixed thanks to #131 & #130
